Tournament.cpp

```diff
@@ -0,0 +1,221 @@
+#include "Tournament.h"
+
+	Tournament::Tournament()
+	{
+		std::string EuroPath = "../FIFApoints/EuroCupParticipants.txt";
```

Comment:
Better pass the paths as arguments, hardcoded paths rarely turn out to be a good idea.